core/utils/pqueue.c

```diff
@@ -25,15 +25,15 @@ int event_matches(void* event1, void* event2) { return (((event_t*)event1)->trig
 
 int reaction_matches(void* a, void* b) { return (a == b); }
 
-pqueue_pri_t get_event_time(void* event) { return (pqueue_pri_t)(((event_t*)event)->time); }
+pqueue_pri_t get_event_time(void* event) { return (pqueue_pri_t)(((event_t*)event)->base.tag.time); }
```

Comment:
This function can be removed.  It's only use (in `environment.c`) can be replaced with the `getpri` function used by the event_q, `pqueue_tag_get_priority`.

Reply:
To remove some more functions e.g., `set_event_position` and `get_event_position`, I made `pqueue_tag_init_customize` more flexible and used it to initialize `recycle_q` in [4b8d42d](https://github.com/lf-lang/reactor-c/pull/390/commits/4b8d42dd65662087ae0b3db77a5cb844ae115446). Does this change make sense?